docs/start/getting-started/fragments/angular/setup.md

```diff
@@ -10,10 +27,10 @@ Now that we have a running Angular app, it's time to set up Amplify for this app
 amplify init
 ```
 
-When you initialize Amplify you'll be prompted for some information about the app.  For newer versions of Angular, you will have to change the Distribution Directory Path from `dist` to `dist/myAmplifyProject` to match how Angular will build your project.
+When you initialize Amplify you'll be prompted for some information about the app.  For newer versions of Angular, you will have to change the Distribution Directory Path from `dist` to `dist/amplifyapp` to match how Angular will build your project.
```

Comment:
"newer versions" -- if possible, get specific. (6+) ?

Reply:
This is intended as a possible troubleshooting. Normally this won't be necessary.